### DISCUSSION TITLE
can we get a scrollbar?

### DISCUSSION THREAD

**Discussion by mcmoebius** [mbnuqw/sidebery#974] [Ideas]
i don't know if i'm missing a setting, but i don't have a scrollbar. can we have one? it's the only thing keeping me from sideberry.

thanks so much for your work on this wonderful extension,
jason m.

**Comment by albino1** [comment]
You should definitely have a scrollbar.

If you're using v5 then the scrollbar settings are right at the top of General:

![sidebery-scrollbars](https://user-images.githubusercontent.com/12164957/222932611-481c688f-120a-4665-a2b4-1bc7cde9ad4f.jpg)

**Reply by mcmoebius** [reply]
thank you!
